config/sets/doctrine-gedmo-annotations-to-attributes.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Php80\Rector\Class_\AnnotationToAttributeRector;
+use Rector\Php80\ValueObject\AnnotationToAttribute;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig
+        ->ruleWithConfiguration(AnnotationToAttributeRector::class, [
+            new AnnotationToAttribute('Gedmo\\Mapping\\Annotation\\Blameable'),
```

Comment:
double `\\` can be changed to single `\`

Reply:
Fixed!